### ISSUE TITLE
Add some basic usage documentation to the README

### ISSUE THREAD

**Issue by bbatsov** [clojure-emacs/orchard#19]
We should probably add a least a couple of sections regarding the high-level usage of `orchard` and the goals of the project.

**Comment by MalloZup** [comment]
autogenerated with https://github.com/MalloZup/doghub:  issue inactive since 450 days. Please update the issue or close it

**Comment by vemv** [comment]
Between https://github.com/clojure-emacs/orchard#design and https://github.com/clojure-emacs/orchard#api-documentation it looks like we're well served :)

cljdoc fix is kind of in its way https://github.com/cljdoc/cljdoc/issues/487

**Comment by bbatsov** [comment]
I meant this ticket more in the sense of explaining what the various APIs are supposed to do with more prose. Something like an overview of namespaces and essential functions, but I guess good ns and function docstrings, plus working cljdoc would be a good enough solution. 

Generally my concern is that few people will understand from the README how are they supposed to use Orchard.

**Comment by Cyrik** [comment]
I agree with @bbatsov. I was confused when I first found orchard through nrpel.
A basic overview of the namespaces plus a few usage examples would be helpful for anyone seeing this for the first time.

**Comment by vemv** [comment]
Alright let's see what we can do :) I agree with `I guess good ns and function docstrings, plus working cljdoc would be a good enough solution.`, in the end Orchard is essentially a bag of defns so there's not that much to understand
